terraform/baremetal-incus/main.tf

```diff
@@ -144,4 +144,8 @@ resource "incus_instance" "instances" {
       "source" = incus_volume.disk3[each.key].name
     }
   }
+
+  lifecycle {
+    ignore_changes = [ running ]
+  }
```

Comment:
What does that do exactly?

Reply:
It prevents Tofu from turning on and off the instances when the state is controlled externally to Tofu.I prefer to do that with different tooling. In my case, I would use the Incus CLI.

The AWS, etc. providers offer the ability to control the on off state of instances, but it is opt in and discouraged.

Ideally the incus provider would work the same way. Then we would not need to disable it.

But what does it do exactly? It tells Tofu to ignore that part of the state file.